package.json

```diff
@@ -98,6 +98,7 @@
     "helper.d.ts"
   ],
   "dependencies": {
+    "fast-safe-stringify": "2.1.1",
```

Comment:
I think we can change the version number format as `^2.1.1`.

Reply:
fixed